public/javascripts/playlist/playlistTable.js

```diff
@@ -71,37 +92,53 @@ function deletePlaylist(name) {
   });
 }
 
-function addPlaylist(args, name, url) {
-  return m
-    .request({
-      method: 'PUT',
-      url: '/user/library',
-      data: {
-        name: name,
-        url: url
-      }
-    })
-    .then(function(res) {
-      if (res.url) {
-        window.location.href = res.url;
-      } else {
+function saveAfterConfirm(args, name) {
+  return new Promise((resolve, reject) => {
+    if (name.indexOf('#') > -1) {
```

Comment:
We can remove this if we've switched to id from name.

Reply:
Yes. I realized after I requested to pull.
So, I will do it.